specification/batch/data-plane/Microsoft.Batch/2017-09-01.6.0/BatchService.json

```diff
@@ -12664,7 +12822,7 @@
         },
         "storageAccountType": {
           "title": "The storage account type to be used for the data disk.",
-          "description": "If omitted, the default is \"Standard_LRS\". Values are:\n\n Standard_LRS - The data disk should use standard locally redundant storage.\n Premium_LRS - The data disk should use premium locally redundant storage.",
+          "description": "If omitted, the default is \"Standard_LRS\".",
```

Comment:
inconsistent. Some places use all lower cases. Here is different.

Reply:
Fixed